### PR TITLE
draft press release

### LINKED ISSUE #29: Write a press release
We need a press release to launch the book properly

### PR DESCRIPTION
**Pull request by mlinksva**
Fixes #29

### CODE REVIEW

**Comment by mlinksva** [comment]
@rejon @barrythrew I have made up quotes for each of you, please edit if you like. And/or we could have quotes from other people.

**Comment by mlinksva** [comment]
I'm not particularly attached to ANYTHING in the draft. Just made up stuff.

There's also a placeholder for the most prestigious places that have written about #freebassel. Suggestions?

File: _includes/download_buttons.html [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
-  <p class="center">
-    <a href="//book.costoffreedom.cc/"><button class="button-download button">HTML</button></a>
-    <a href="https://www.gitbook.com/download/pdf/book/costoffreedom/costoffreedom-book"><button class="button-download button">PDF</button></a>
-    <a href="https://www.gitbook.com/download/epub/book/costoffreedom/costoffreedom-book"><button class="button-download button">EPUB</button></a>
+    <span class="nine columns">
+      Read The Book
+      <br>
+      <br>
+      <a href="//book.costoffreedom.cc/"><button class="button-download button">HTML</button></a>
+      <a href="https://www.gitbook.com/download/pdf/book/costoffreedom/costoffreedom-book"><button class="button-download button">PDF</button></a>
```
Comment (mlinksva) [inline _includes/download_buttons.html]: @clemsos should this point to https://github.com/costoffreedom/costoffreedom-book/raw/master/costoffreedom-final.pdf instead? Or maybe rather the file with a more canonical name (like say costoffreedom-edition-1.1.pdf on the website?) The LaTeX generated PDF looks a ton better, ideally nobody ever sees the gitbook generated one, right?

Comment (clemsos) [inline _includes/download_buttons.html]: Yes, you're right. should ttly point to the latex pdf

**Comment by hellekin** [comment]
Maybe mention where the proceeds of the book sales go.

**Pull request merged**